src/routes/MyPage.jsx

```diff
@@ -0,0 +1,288 @@
+import { useEffect, useState } from "react";
+import { Link } from "react-router-dom";
+import { SmallPost } from "../components/Posts";
+import {
+  getPosts,
+  getTags,
+  getUserInMyPage,
+  updateUserInMyPage,
+} from "../apis/api";
+import { getCookie } from "../utils/cookie";
+
+const MyPage = () => {
+  const [postList, setPostList] = useState([]);
+  const [profile, setProfile] = useState({
+    id: "",
+    user: {},
+    college: "",
+    major: "",
+  });
```

Comment:
사실 우리에게 필요한건 email, username, college, major기 떄문에

```suggestion
  const [profile, setProfile] = useState({
    email: "",
    username: "",
    college: "",
    major: "",
  });
```

이런식으로 할 것같아요!
그리고 사실 Js에서는 상관없지만 typescript에서는 `id`는 string이 아니기때문에 초기값을 ""와 같이 설정하면 안됩니다!